docs/howtoguides/enable_cis.md

```diff
@@ -0,0 +1,18 @@
+# How to enable CIS
+
+To access the CIS tooling first enable the software repository.
```

Comment:
Only for Xenial/Bionic. On Focal, we should point to USG in this doc

Reply:
Good point. I have updated the doc, but let me know if you think we should have a bigger warning there

Reply:
The note is fine I guess - we can probably (in next iterations maybe, no need to do it right now) include a `enable_usg` document like this one.